shell/src/test/java/alluxio/cli/fsadmin/command/SummaryCommandTest.java

```diff
@@ -41,14 +42,14 @@
 
   @Before
   public void prepareDependencies() throws IOException {
-    // We generate random values for MasterInfo and BlockMasterInfo
+    // Generate random values for MasterInfo and BlockMasterInfo
     // Prepare mock meta master client
     mMetaMasterClient = Mockito.mock(MetaMasterClient.class);
     MasterInfo masterInfo = new MasterInfo()
         .setMasterAddress("testAddress")
         .setWebPort(1231)
         .setRpcPort(8462)
-        .setStartTimeMs(213124234312231L)
+        .setStartTimeMs(1131242343122L)
```

Comment:
Could we actually randomly generate all the values (random strings, random ints, random timestamps, etc), save the instance, and then later, use that instance to generate the expected string outputs?

Reply:
Andrew and I want to do explicit check. I first use random to generate random values, then the expected output I use the same method to generate as what I use in summaryCommand.
Like 

long uptime = random.nextLong()
expectedOutput = "Uptime: " + CommonUtils.convertMsToClockTime(masterInfo.getUpTimeMs()

Using the same way as what I used in summaryCommand, so just like check nothing.

Reply:
Ok. We can change it later if we want.